contracts/LockedNORI.sol

```diff
@@ -675,6 +675,16 @@ contract LockedNORI is
     return ERC777Upgradeable.authorizeOperator(operator);
   }
 
+  /**
+   * @dev This is a wrapper, minting is only supported via `send`
+   *
+   * See {ERC20-_mint}.
+   *
+   */
+  function mint(address, uint256) public virtual override {
```

Comment:
I believe with my latest this is also redundant as the public `mint` function is now longer visible by default and so this just exposes it for no reason

(mint was previously public when we were inheriting from the ERC20 preset)